get_who_theme.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+
+import pandas as pd
+import requests
+
+
+def main():
+    # Run like `python get_who_theme.py "Overweight and Obesity"`
+    parser = argparse.ArgumentParser()
+    parser.add_argument("theme_name")
+    args = parser.parse_args()
+
+    requested_indicators = pd.read_csv("requestedIndicators.csv")
+
+    theme = get_theme(args.theme_name, requested_indicators)
+    transform_theme(theme)
+    write_excel(theme)
+    write_csv(theme)
+
+
+def get_theme(theme_name: str, requested_indicators: pd.DataFrame) -> pd.DataFrame:
```

Comment:
I added lightweight type annotations because I thought they might aid understanding. They're not necessary.